sycl/include/sycl/ext/oneapi/matrix/static-query.hpp

```diff
@@ -5,475 +5,604 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 // ===--------------------------------------------------------------------=== //
+// This file implements the static query interface for the joint_matrix
```

Comment:
Sorry, I meant to add a description to this PR to describe what this patch does ([to here](https://github.com/intel/llvm/pull/4510#issue-728870075))

Reply:
done